roles/nuage_node/vars/main.yaml

```diff
@@ -23,3 +23,5 @@ cni_conf_dir: "/etc/cni/net.d/"
 cni_bin_dir: "/opt/cni/bin/"
 
 nuage_plugin_crt_dir: /usr/share/vsp-openshift
+openshift_atomic_node_config_file: /etc/sysconfig/atomic-openshift-node
```

Comment:
just to make sure, can we use the same variable name that service file is using?

`/etc/sysconfig/{{ openshift.common.service_type }}-node`